components/modal/src/ModalFooter.js

```diff
@@ -0,0 +1,20 @@
+/** @jsx jsx */
+
+import React from 'react';
+import { jsx, useTheme } from '@westpac/core';
+
+export const ModalFooter = props => {
+	const { modal } = useTheme();
+
+	return (
+		<div
+			css={{
+				backgroundColor: modal.footer.backgroundColor,
+				borderTop: `${modal.footer.borderWidth} solid ${modal.footer.borderColor}`,
+				textAlign: 'right',
+				padding: modal.footer.padding,
+			}}
```

Comment:
Need to style adjacent buttons in the footer... they shouldn't touch

Reply:
How much distance? GEL 2 has 2px spacing but it seems small...

Reply:
Also do we think this is necessary? Modal Footer is just a wrapper for styling purposes. 
Should we just let the devs implement the button spacing since we dont know how many buttons they will put in the footer?

Reply:
Yeah we need to mandate and provide this styling or it will likely be implemented inconsistently 

Buttons are inline-block and in GUI2.0 we're assuming there is a space character in-between which we don't have to worry with React/jsx 🙌

Should be 6px gap

```
& + & {
	marginLeft: 6,
}
```